patches/chrome-installer-util-util_constants.cc.patch

```diff
@@ -1,8 +1,22 @@
 diff --git a/chrome/installer/util/util_constants.cc b/chrome/installer/util/util_constants.cc
-index 4ed3469f1dae894df6636185414993c657b7b2c4..1ff629ffcd6513fb2d65785be9b48aec2d326731 100644
+index 4ed3469f1dae894df6636185414993c657b7b2c4..76da2acd8391b4ae3f6d2efe2e8eaaec7539251f 100644
 --- a/chrome/installer/util/util_constants.cc
 +++ b/chrome/installer/util/util_constants.cc
-@@ -187,7 +187,11 @@ const char kGoogleUpdateIsMachineEnvVar[] = "GoogleUpdateIsMachine";
+@@ -171,7 +171,13 @@ namespace env_vars {
+ // The presence of this environment variable with a value of 1 implies that
+ // setup.exe should run as a system installation regardless of what is on the
+ // command line.
++#if defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD)
++// Reflect brave branded-omaha variable name.
++// Company name(BraveSoftware) + Product name(Update).
++const char kGoogleUpdateIsMachineEnvVar[] = "BraveSoftwareUpdateIsMachine";
```

Comment:
is this necessary? I think it would be better to have consistent behavior for this env var with Chrome

Reply:
This var name comes from omaha.
To customize omaha for brave, company name should be changed to `BraveSoftware` because it is related with install directory. Then omaha makes env var by using it.
So, I think we should follow omaha var name in here.

Reply:
can be done with the later string substitution method.